<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VMLBR8N" failures="1" tests="1" name="ClientTest" time="52.548" errors="0" timestamp="2021-08-16T03:08:13 GMT-07:00">
  <testcase classname="testScenario.Client.ClientTest01" name="clientTest" time="31.459">
    <failure type="java.lang.AssertionError" message="expected [3] but found [No results found.]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [No results found.]
at utilPack.BasePge.validateText(BasePge.java:80)
at testPage.MavenPageObject.Clients.searchClientDetails(Clients.java:51)
at testScenario.Client.ClientTest01.clientTest(ClientTest01.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- clientTest -->
</testsuite> <!-- ClientTest -->
